### PR TITLE
Fix cellVdec regression to be harmonized with the new interrupts model

### PR DESCRIPTION
**Pull request by elad335**

### CODE REVIEW

File: rpcs3/Emu/Cell/Modules/cellAudio.cpp [hunk 1]
```diff
@@ -1145,7 +1145,6 @@ error_code cellAudioQuit()
 	}
 
 	// TODO
-	g_audio.keys.clear();
```
Comment (Megamouse) [inline rpcs3/Emu/Cell/Modules/cellAudio.cpp:1148]: Please add a comment to not clear it for the next person who thinks it's missing.

Comment (Megamouse) [reply rpcs3/Emu/Cell/Modules/cellAudio.cpp:1148]: Also, just wondering, shouldn't the queue still be deleted in the removequeue function? I think it's missing

**Comment by Megamouse** [comment]
Should fix #11761

File: rpcs3/Emu/Cell/Modules/cellAudio.cpp (hunk 1 above)
Comment (elad335) [reply rpcs3/Emu/Cell/Modules/cellAudio.cpp:1148]: Nope, I was fool, you were right.

**Comment by Megamouse** [comment]
Can you detach the audio fix?
It's unrelated to the hotfix and it would be better for regression testing if it were two separate prs.
(You could reopen the PR i closed earlier)

**Pull request merged**